database/cassandra/cassandra.go

```diff
@@ -155,9 +157,24 @@ func (c *Cassandra) Run(migration io.Reader) error {
 	}
 	// run migration
 	query := string(migr[:])
-	if err := c.session.Query(query).Exec(); err != nil {
-		// TODO: cast to Cassandra error and get line number
-		return database.Error{OrigErr: err, Err: "migration failed", Query: migr}
+
+	if c.config.MultiStatementEnabled {
+		// split query by semi-colon
+		queries := strings.Split(query, ";")
+
+		for _, q := range(queries) {
+			tq := strings.TrimSpace(q)
+			if (tq == "") { continue }
+			if err := c.session.Query(tq).Exec(); err != nil {
+				// TODO: cast to Cassandra error and get line number
+				return database.Error{OrigErr: err, Err: "migration failed", Query: migr}
+			}
+		}
+	} else {
```

Comment:
`return` in the `if` block above and remove the `else` to save a level of indentation
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow